app/mailers/application_mailer.rb

```diff
@@ -1,4 +1,4 @@
 class ApplicationMailer < ActionMailer::Base
-  default from: 'from@example.com'
+  default from: 'appointments@pensionwise.gov.uk'
```

Comment:
Can we add the sender name here too? In the form `Pension Wise Appointments <appointments@pensionwise.gov.uk>`